lib/WorseReflection/Core/SourceCodeLocator/InternalStubs/Prophecy.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Prophecy\Prophecy;
+
+/**
+ * @template T of object
+ * @template-implements ProphecyInterface<T>
+ */
+class ObjectProphecy implements ProphecyInterface
+{
+    /**
+     * @return T
+     */
+    public function reveal()
+    {
+    }
+}
+
```

Comment:
probably need to add custom stubs for Prophecy, and enable stubs to be added by extensions.